_episodes_rmd/11-supp-read-write-csv.Rmd

```diff
@@ -115,6 +115,19 @@ carSpeeds$Color
 ```
 
 That's better! And we can see how the data now is read as character instead of factor.
+ 
+ 
+ 
```

Comment:
Do we need this many newlines above & below? I think 1 or 2 are enough separators.